### ISSUE TITLE
I get an error with create a DM

### ISSUE THREAD

**Issue by excelant100** [Merubokkusu/Discord-S.C.U.M#29]
ok so I used the example to fetsh members wich is working correctly. I get all their ids and using for loop I want to create a DM channel foreach of them. 

`@bot.gateway.command
def memberTest(resp):
	guild_id = '714338144954351669'
	channel_id = '743719231274876998'
	if resp.event.ready_supplemental:
		bot.gateway.fetchMembers(guild_id, channel_id)
	if bot.gateway.finishedMemberFetching(guild_id):
		bot.gateway.removeCommand(memberTest)
		bot.gateway.close()

bot.gateway.run()
print('aa')
for member in bot.gateway.session.guild('714338144954351669').members:
    bot.createDM([member])`
    
    When I replace member with a manual id its working fine.

**Comment by excelant100** [comment]
ok in this case if I use Discum to fetsh members and use discord py to send message. I wont get flagged that quick since its normal for bots to send a lot of DMs at a time?

**Comment by Merubokkusu** [comment]
So discord.py is for Bots, discum if focused on userbots (which are are against the tos)
You can get server members with discord.py if you’re making a bot 
https://discordpy.readthedocs.io/en/latest/api.html?highlight=get%20all%20members#discord.Client.get_all_members

**Comment by excelant100** [comment]
yeah I know this, the goal is to DM spam. What I want to do is simple. My userbot will join random discord server and spam people there. Since DMing with discum is risky, I though that using discord py to actually  DM the user would be better, but I need discum to fetsh members, I wont have to write a command so the bot start spamming. I want it that once it joined the server, after some delay, it gets to works. I dont think that only discord py will let me achieve this, so I need both. Btw you can still use discord py for userbots.

**Comment by excelant100** [comment]
my account to got flagged with createDM. Gonna make a new one through a VPN, I hope they wont ask for phone verification.

**Comment by Merubokkusu** [comment]
I mean, no matter what you'll get banned for mass DM spamming.
Discord already sends you a cool down warning for every 10 DM's sent within 10 minutes.
SMS verification could delay it, but overall its ticking time bomb.

**Comment by excelant100** [comment]
I wont spam like this, ill put reasonable delay. like a random delay between 3 minutes or 7 to send 1 DM.  I know I will get banned, in this case just create another account. I want to like send around 500 to 600 Dms per day per account. Btw, bots like mee6 etc. send more than 10 Dms per second. I dont care if they get ban, ill just make another (yeah ill buy some proxy and 2captcha things) Oh between is there a function do automatic sms verification.

ill try to make money of this of course.

**Comment by Merubokkusu** [comment]
Mee6 is a server bot.
There hasn’t been much testing for that length of time, but I don’t see the point of using two different libraries

**Comment by excelant100** [comment]
so you mean using discord py send message, will have the same effect that using discum create DM. I wont get flagged faster using Discum. like discum does not the the same request than discord py. discum do the same request that a user do when using discord right? you check the network thing and simulate the request. But I dont think python py does the same thing, since its made for discord bots. So maybe its safer to spam send DM with discord bot than discum. I am not sure if you understand what I meant.

**Comment by Merubokkusu** [comment]
I feel like discum would be safer as it uses the same requests as the actual client uses, there’s room to experiment with both and see the outcome.

But using two different libraries will definitely speed up getting banned .

**Comment by excelant100** [comment]
yeah youre right I guess, ill test and see how it will go. I hope it can work the way I am thinking.
